src/stylesheets/core/_variables.scss

```diff
@@ -606,19 +618,19 @@ settings
 // Removing the !default from $em-base so we are not inheriting that value from Bourbon.
 
 $em-base:             $theme-root-font-size;
-$base-font-size:      1.7rem !default;
-$small-font-size:     1.4rem !default;
-$lead-font-size:      2rem !default;
-$title-font-size:     5.2rem !default;
-$h1-font-size:        4rem !default;
-$h2-font-size:        3rem !default;
-$h3-font-size:        2rem !default;
-$h4-font-size:        1.7rem !default;
-$h5-font-size:        1.5rem !default;
-$h6-font-size:        1.3rem !default;
-$base-line-height:    1.5 !default;
-$heading-line-height: 1.3 !default;
-$lead-line-height:    1.7 !default;
+$base-font-size:      $theme-base-font-size;
+$small-font-size:     $theme-small-font-size;
+$lead-font-size:      $theme-lead-font-size;
+$title-font-size:     $theme-title-font-size;
+$h1-font-size:        $theme-h1-font-size;
+$h2-font-size:        $theme-h2-font-size;
+$h3-font-size:        $theme-h3-font-size;
+$h4-font-size:        $theme-h4-font-size;
+$h5-font-size:        $theme-h5-font-size;
+$h6-font-size:        $theme-h6-font-size;
+$base-line-height:    $theme-base-line-height;
+$heading-line-height: $theme-heading-line-height;
+$lead-line-height:    $theme-lead-line-height;
 
```

Comment:
Actually, I should revert these to the old values for now — to allow us to more specifically scope our work. Globally changing these vars will cause unexpected problems right now.

PLEASE HOLD ON MERGING UNTIL I FIX THIS